versioncontrol/peer.go

```diff
@@ -39,6 +38,7 @@ type Config struct {
 }
 
 // ServiceVersions provides a list of allowed Versions per Service.
+// TODO: rename to OldVersionsConfig?
```

Comment:
Why `Old..`?

Reply:
This config is only used by `OldMinimum`, hence "old". It too should be deprecated and removed eventually.